client/lib/user/user.js

```diff
@@ -23,7 +28,7 @@ import { getLanguage } from 'lib/i18n-utils/utils';
 import localforage from 'lib/localforage';
 import { getActiveTestNames, ABTEST_LOCALSTORAGE_KEY } from 'lib/abtest/utility';
 
-/**
+/*
```

Comment:
why did we remove the JSDoc marker here?

Reply:
ESLint was complaining about missing docs for params and return values. @brandonpayton eventually fixed the warnings much better than I did. Thanks!